msg-socket/src/rep/socket.rs

```diff
@@ -121,7 +122,12 @@ where
             return Err(RepError::NoValidEndpoints);
         };
 
-        debug!("Listening on {:?}", local_addr);
+        let id = IdBase58::new();
```

Comment:
I understand the use of an ID here to associate the spans to a certain driver, but since `local_addr` is already unique to a driver, can we just use that? The IDs are quite verbose in stdout.

Reply:
Right, for `ReqSocket` this makes sense!